src/material-experimental/mdc-typography/_all-typography.scss

```diff
@@ -1,5 +1,52 @@
+@import '@material/typography/variables.import';
 @import '../mdc-theming/all-theme';
 
+@function mat-mdc-typography-config(
```

Comment:
Add a SassDoc for this function?

(I know we haven't done this much in the past, but I'd like to start)